docs/ref/settings.txt

```diff
@@ -775,6 +775,9 @@ database into an in-memory JSON string before running tests (used to restore
 the database state between tests if you don't have transactions). You can set
 this to ``False`` to speed up creation time if you don't have any test classes
 with :ref:`serialized_rollback=True <test-case-serialized-rollback>`.
+It should also not be set to ``False`` if want to use :option:`test --keepdb`
```

Comment:
`It should also not be set to...` → `Do not set this to...`
`...if want to use...` → `...if you want to use...`